<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>按下键盘0~6旋转体绘制材质</string>
  </property>
  <widget class="QVTKWidget" name="volumeWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>735</width>
     <height>365</height>
    </rect>
   </property>
  </widget>
  <widget class="QVTKWidget" name="sagitalWidget">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>55</y>
     <width>735</width>
     <height>365</height>
    </rect>
   </property>
  </widget>
  <widget class="QVTKWidget" name="coronalWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>455</y>
     <width>735</width>
     <height>365</height>
    </rect>
   </property>
  </widget>
  <widget class="QVTKWidget" name="axialWidget">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>455</y>
     <width>735</width>
     <height>365</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>医疗规划系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="navigationButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>3</y>
     <width>120</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="hintLabel">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>5</y>
     <width>750</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1530</x>
     <y>3</y>
     <width>60</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="greenButton1">
   <property name="geometry">
    <rect>
     <x>1531</x>
     <y>55</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="greenButton2">
   <property name="geometry">
    <rect>
     <x>1531</x>
     <y>115</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="shrinkButton">
   <property name="geometry">
    <rect>
     <x>1531</x>
     <y>325</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="magnifyButton">
   <property name="geometry">
    <rect>
     <x>1531</x>
     <y>265</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="translateButton">
   <property name="geometry">
    <rect>
     <x>1531</x>
     <y>175</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="coronalSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>830</y>
     <width>735</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="axialSlider">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>830</y>
     <width>735</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="sagitalSlider">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>425</y>
     <width>735</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>425</y>
     <width>735</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <zorder>greenButton1</zorder>
  <zorder>volumeWidget</zorder>
  <zorder>sagitalWidget</zorder>
  <zorder>coronalWidget</zorder>
  <zorder>axialWidget</zorder>
  <zorder>logoLabel</zorder>
  <zorder>navigationButton</zorder>
  <zorder>hintLabel</zorder>
  <zorder>exitButton</zorder>
  <zorder>greenButton2</zorder>
  <zorder>shrinkButton</zorder>
  <zorder>magnifyButton</zorder>
  <zorder>translateButton</zorder>
  <zorder>coronalSlider</zorder>
  <zorder>axialSlider</zorder>
  <zorder>sagitalSlider</zorder>
  <zorder>volumeSlider</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
